ledger/store/trackerdb/data.go

```diff
@@ -90,21 +90,24 @@ type ResourcesData struct {
 	_struct struct{} `codec:",omitempty,omitemptyarray"`
 
 	// asset parameters ( basics.AssetParams )
-	Total         uint64         `codec:"a"`
-	Decimals      uint32         `codec:"b"`
-	DefaultFrozen bool           `codec:"c"`
-	UnitName      string         `codec:"d"`
-	AssetName     string         `codec:"e"`
-	URL           string         `codec:"f"`
-	MetadataHash  [32]byte       `codec:"g"`
-	Manager       basics.Address `codec:"h"`
-	Reserve       basics.Address `codec:"i"`
-	Freeze        basics.Address `codec:"j"`
-	Clawback      basics.Address `codec:"k"`
+	Total           uint64         `codec:"a"`
+	Decimals        uint32         `codec:"b"`
+	DefaultFrozen   bool           `codec:"c"`
+	UnitName        string         `codec:"d"`
+	AssetName       string         `codec:"e"`
+	URL             string         `codec:"f"`
+	MetadataHash    [32]byte       `codec:"g"`
+	Manager         basics.Address `codec:"h"`
+	Reserve         basics.Address `codec:"i"`
+	Freeze          basics.Address `codec:"j"`
+	Clawback        basics.Address `codec:"k"`
+	GlobalFrozen    bool           `codec:"A"`
+	LastAssetFreeze uint64         `codec:"B"`
 
 	// asset holding ( basics.AssetHolding )
-	Amount uint64 `codec:"l"`
-	Frozen bool   `codec:"m"`
+	Amount            uint64 `codec:"l"`
+	Frozen            bool   `codec:"m"`
+	LastAccountFreeze uint64 `codec:"C"`
```

Comment:
This needs to go down below so that people see the "C" when they consider adding a new field.